components/server/src/server.rs

```diff
@@ -1465,8 +1461,13 @@ where
                         capacity
                     );
                 }
-                // Update disk status.
-                disk::set_disk_status(cur_disk_status);
+                // Update disk status if disk space checker is enabled.
+                if reserve_space == 0 && reserve_raft_space == 0 {
+                    info!("disk space checker not enabled");
```

Comment:
nit: The log seems no longer accurate now since the disk space checker is always enabled. Maybe something like "ignoring disk status update as no reserve space is set."?